jina/clients/__init__.py

```diff
@@ -126,15 +126,15 @@ def update(
         on_error: CallbackFnType = None,
         on_always: CallbackFnType = None,
         **kwargs
-    ) -> Awaitable:
+    ) -> Awaitable[list]:
```

Comment:
capital Letter. Do we know whay is the list of? is a list of Documents?

Reply:
oh yeah, a list of responses